stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/global/internal/GlobalMutableState.kt

```diff
@@ -215,8 +215,14 @@ public class GlobalMutableState private constructor(
 
     override fun tryEmitTypingEvent(cid: String, typingEvent: TypingEvent) {
         _typingChannelsOld.tryEmit(typingEvent)
+
         val typingChannelsCopy = _typingChannels.value.toMutableMap()
-        typingChannelsCopy[cid] = typingEvent
+
+        if (typingEvent.users.isEmpty()) {
```

Comment:
Previously we haven't removed keys for typing events that contained no active typing events.
This aims to fix that and not populate the map with keys that contain empty lists.